gitops/argocd/tektoncd/tekton-config.yaml

```diff
@@ -11,5 +16,6 @@ spec:
     resources:
       - taskrun
     keep: 100
```

Comment:
Why a quantity rather than a duration specified with [keep-since](https://github.com/tektoncd/operator/blob/main/pkg/apis/operator/v1alpha1/tektonconfig_types.go#L58-L61)?
A quantity makes capacity management easier but a duration provides a better UX with a predictable behavior.
Any special reasoning for deciding on 100? It seems to me a bit low, although this would need to be configured per env.

Reply:
Umm, there is no particular reason, "why 100?", It is actually the default TektonConfig, refer https://github.com/tektoncd/operator/blob/main/config/crs/kubernetes/config/all/operator_v1alpha1_config_cr.yaml#L26, this PR only modifies the types of resources to be pruned, everything else is kept as it is. Indeed a duration will be more helpful, should I change that too?